libcontainer/rootfs_linux.go

```diff
@@ -470,8 +484,35 @@ func mountToRootfs(c *mountConfig, m mountEntry) error {
 		if err := prepareBindMount(m, rootfs); err != nil {
 			return err
 		}
-		if err := mountPropagate(m, rootfs, mountLabel); err != nil {
-			return err
+
+		if m.IsBind() && m.IsIDMapped() {
+			if m.idmapFd == nil {
+				return fmt.Errorf("error creating mount %+v: idmapFd is nil, should point to a valid fd", m)
+			}
+			if err = unix.MoveMount(*m.idmapFd, "", -int(unix.EBADF), dest, unix.MOVE_MOUNT_F_EMPTY_PATH); err != nil {
```

Comment:
This is a weird use of an errno value. From what I understand, `dest` is absolute path and so `destFd` argument is ignored. You can probably just set it to -1.

Reply:
This was solved already :)